src/Microsoft.Health.Dicom.Core/Features/Store/StoreValidationResultBuilder.cs

```diff
@@ -62,23 +53,18 @@ public void Add(ValidationWarnings warningCode, DicomTag dicomTag = null)
         }
     }
 
-    /// <summary>
-    /// Adds a tag to a list representing invalid Dicom items.
-    /// </summary>
-    /// <param name="tag">Invalid item's tag to add.</param>
-    public void AddInvalidTag(DicomTag tag)
-    {
-        _invalidDicomTags.Add(tag);
-    }
-
     private static string GetFormattedText(string message, DicomTag dicomTag = null)
     {
         EnsureArg.IsNotNull(message, nameof(message));
 
         if (dicomTag == null)
             return message;
 
-        return $"{dicomTag} - {dicomTag.DictionaryEntry.Keyword} - {message}";
+        return string.Format(
```

Comment:
@jovinson-ms, Now the error message will look like 
`DICOM100: Dicom element 'StudyDate' failed validation for VR 'DA': Value cannot be parsed as a valid date.`

Do you think having the Tag code in the error help the users to fix them quickly?

Reply:
Changed back to `DICOM100: (0008,0020) - Content "NotAValidStudyDate" does not validate VR DA: one of the date values does not match the pattern YYYYMMDD`